spec/support/feature_helpers.rb

```diff
@@ -245,10 +252,12 @@ def complete_login_password_screen(password)
 end
 
 def complete_reset_password_screen(password=nil)
+  expect(page.current_path).to eq(password_reset_path)
   password ||= 'Passw0rd!'
-  fill_in (t :"identities.set.password"), with: password
-  fill_in (t :"identities.set.confirm_password"), with: password
+  fill_in 'set_password_password', with: password
```

Comment:
just so I know - what are you using as the selector here? The form field name?

Reply:
The form field id